CONTRIBUTING.md

```diff
@@ -89,15 +89,15 @@ If you are going to be contributing back to InfluxDB please take a second to sig
 
 ### Installing Go
 
-InfluxDB requires Go 1.17.
+InfluxDB requires Go 1.18.
 
 At InfluxData we find `gvm`, a Go version manager, useful for installing Go.
 For instructions on how to install it see [the gvm page on github](https://github.com/moovweb/gvm).
 
 After installing gvm you can install and set the default go version by running the following:
 ```bash
-$ gvm install go1.17
-$ gvm use go1.17 --default
+$ gvm install go1.18
+$ gvm use go1.18 --default
```

Comment:
It would _probably_ be better to put 1.18.1 here since that's what we're using for builds, but this at least gets it from not building to building